public/pages/CreateDetector/components/DefineDetector/components/DetectorDetails/DetectorBasicDetailsForm.tsx

```diff
@@ -69,12 +94,17 @@ export default class DetectorBasicDetailsForm extends Component<
         <EuiSpacer size={'m'} />
 
         {/*// TODO: Implement regex pattern validation for description field.*/}
-        <EuiFormRow label={<FormFieldHeader headerTitle={'Description'} optionalField={true} />}>
+        <EuiFormRow
+          label={<FormFieldHeader headerTitle={'Description'} optionalField={true} />}
+          isInvalid={descriptionFieldTouched && descriptionIsInvalid}
+          error={getNameErrorMessage(detectorName, nameIsInvalid, nameFieldTouched)}
+        >
           <EuiTextArea
             placeholder={'Enter a description for the detector.'}
             compressed={true}
             value={detectorDescription}
-            onChange={onDetectorInputDescriptionChange}
+            onBlure={this.onDescriptionBlur}
+            onChange={this.onDescriptionChange}
```

Comment:
I think you also want `error={getDescriptionErrorMessage(...)}` here

Reply:
We call that here https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/84/files#diff-ce777ba621c03240471af2cfaa5a09eb86107b8ed94fd28a033b6913b2ef3515R100